content/docs/5.1/deploy/install/prerequisites.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Prerequisites
+description: Learn the software and hardware recommendations for deploying and running TiKV
+menu:
+    "5.1":
+        parent: Install TiKV
+        weight: 1
+---
+As an open source distributed key-value database, TiKV can be deployed in the Intel architecture server, ARM architecture server, and major virtualization environments and runs well. TiKV supports most of the major hardware networks and Linux operating systems.
+
+## Linux OS version requirements
+
+|    Linux OS Platform     |          Version          |
+|:------------------------:|:-------------------------:|
+| Red Hat Enterprise Linux | 7.3 or later 7.x releases |
+|          CentOS          | 7.3 or later 7.x releases |
+| Oracle Enterprise Linux  | 7.3 or later 7.x releases |
+|        Ubuntu LTS        |      16.04 or later       |
+
+Other Linux OS versions such as Debian Linux and Fedora Linux might work but are not officially supported.
+
+
+## Software recommendations
+
+### Control machine
+
+| Software | Version        |
+|:-------- |:-------------- |
+| sshpass  | 1.06 or later  |
+| TiUP     | 1.4.0 or later |
+
+{{< info >}}
+It is required that you [deploy TiUP on the control machine](../production#step-1-install-tiup-on-the-control-machine) to operate and manage TiKV clusters.
+{{< /info >}}
+
+### Target machines
+
+| Software | Version         |
+|:-------- |:--------------- |
+| sshpass  | 1.06 or later   |
+| numa     | 2.0.12 or later |
+| tar      | any             |
+
+## Server recommendations
+
+You can deploy and run TiKV on the 64-bit generic hardware server platform in the Intel x86-64 architecture or on the hardware server platform in the ARM architecture. The requirements and recommendations about server hardware configuration (ignoring the resources occupied by the operating system itself) for development, test, and production environments are as follows:
+
+### Development and test environments
+
+| Component |   CPU   | Memory | Local Storage |       Network        | Instance Number (Minimum Requirement) |
+|:---------:|:-------:|:------:|:-------------:|:--------------------:|:-------------------------------------:|
+|    PD     | 4 core+ | 8 GB+  | SAS, 200 GB+  | Gigabit network card |                   1                   |
+|   TiKV    | 8 core+ | 32 GB+ | SAS, 200 GB+  | Gigabit network card |                   3                   |
```

Comment:
Why 3 TiKV is needed?

Reply:
At least 3 TiKV is needed for production environments.

Reply:
For performance-related test, the benchmark will be inaccurate if using only one TiKV.